macx/scripts/osxdist.py

```diff
@@ -325,37 +324,37 @@ def package_server():
 	name = 'Murmur-OSX-Static-%s' % ver
 
 	# Fix .ini files
-	p = Popen(('bash', 'mkini.sh'), cwd='scripts')
+	p = Popen(('bash', 'mkini.sh'), cwd=os.path.join(options.source_dir, 'scripts'))
 	retval = p.wait()
 	if retval != 0:
-		raise Exception('build-overlay-installer failed')
+		raise Exception('mkini.sh failed')
 
-	destdir = os.path.join('release', name)
+	destdir = os.path.join(options.binary_dir, name)
 	if os.path.exists(destdir):
 		shutil.rmtree(destdir)
 	os.mkdir(destdir)
 
-	shutil.copy('installer/gpl.txt', os.path.join(destdir, 'LICENSE'))
-	shutil.copy('README.static.osx', os.path.join(destdir, 'README'))
-	shutil.copy('CHANGES', os.path.join(destdir, 'CHANGES'))
-	shutil.copy('scripts/murmur.pl', os.path.join(destdir, 'murmur.pl'))
-	shutil.copy('scripts/weblist.pl', os.path.join(destdir, 'weblist.pl'))
-	shutil.copy('scripts/icedemo.php', os.path.join(destdir, 'icedemo.php'))
-	shutil.copy('scripts/weblist.php', os.path.join(destdir, 'weblist.php'))
-	shutil.copy('scripts/murmur.ini.osx', os.path.join(destdir, 'murmur.ini'))
-	shutil.copy('src/murmur/Murmur.ice', os.path.join(destdir, 'Murmur.ice'))
-
-	shutil.copy('release/murmurd', os.path.join(destdir, 'murmurd'))
+	shutil.copy(os.path.join(options.binary_dir, 'installer/gpl.txt'), os.path.join(destdir, 'LICENSE'))
+	shutil.copy(os.path.join(options.binary_dir, 'README.static.osx'), os.path.join(destdir, 'README'))
+	shutil.copy(os.path.join(options.binary_dir, 'CHANGES'), os.path.join(destdir, 'CHANGES'))
+	shutil.copy(os.path.join(options.binary_dir, 'scripts/murmur.pl'), os.path.join(destdir, 'murmur.pl'))
+	shutil.copy(os.path.join(options.binary_dir, 'scripts/weblist.pl'), os.path.join(destdir, 'weblist.pl'))
+	shutil.copy(os.path.join(options.binary_dir, 'scripts/icedemo.php'), os.path.join(destdir, 'icedemo.php'))
+	shutil.copy(os.path.join(options.binary_dir, 'scripts/weblist.php'), os.path.join(destdir, 'weblist.php'))
+	shutil.copy(os.path.join(options.binary_dir, 'scripts/murmur.ini.osx'), os.path.join(destdir, 'murmur.ini'))
+	shutil.copy(os.path.join(options.binary_dir, 'src/murmur/Murmur.ice'), os.path.join(destdir, 'Murmur.ice'))
+
+	shutil.copy(os.path.join(os.path.join(options.binary_dir, 'murmurd'), os.path.join(destdir, 'murmurd')))
```

Comment:
```suggestion
	shutil.copy(os.path.join(options.binary_dir, 'murmurd'), os.path.join(destdir, 'murmurd'))
```
I think you used an additional `os.path.copy` here that accidentally wrapped the whole argument (which is why LGTM complained)

Reply:
Thank you, fixed.